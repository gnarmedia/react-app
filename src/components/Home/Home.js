/**
 * @file Home component
 */

import React from "react";

/**
 * Renders Home component.
 *
 * @example
 * <Home />
 *
 * @returns {Function} Home component.
 */
const Home = () => (
  <>
    <section className="hero">
      <div className="hero-body">
        <h1 className="title">Welcome to React with serverside rendering!</h1>
        <h2 className="subtitle">...now to add all the other stuff...</h2>
      </div>
    </section>
  </>
);

export default Home;
