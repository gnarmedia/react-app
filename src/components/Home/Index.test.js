/**
 * @file Sign Up component tests
 */

import React from "react";
import { render } from "enzyme";

import Home from "./Home";

describe("Home", () => {
  it("should render the Home component", () => {
    const wrapper = render(<Home />);

    expect(wrapper).toMatchSnapshot();
  });
});
