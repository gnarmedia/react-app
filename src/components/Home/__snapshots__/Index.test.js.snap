// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render the Home component 1`] = `
<section
  class="hero"
>
  <div
    class="hero-body"
  >
    <h1
      class="title"
    >
      Welcome to React with serverside rendering!
    </h1>
    <h2
      class="subtitle"
    >
      ...now to add all the other stuff...
    </h2>
  </div>
</section>
`;
